frontend/src/tests/components/Commons/ProfitsTable.test.js

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+import ProfitsTable from "main/components/Commons/ProfitsTable"; 
+import profitsTableFixtures from "fixtures/profitsTableFixtures";
+
+describe("Profits tests", () => {
+
+    test("renders without crashing", () => {
+        render(
+            <ProfitsTable profits={profitsTableFixtures.threeFormattedProfits} />
```

Comment:
The variable names are very intuitive and comprehensive. Makes the code more readable!

Reply:
Thanks!